src/component/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Display(props) {
+  const { result } = props;
+
+  return (
+    <div>
+      <h3>Display Component</h3>
+      <p>{result}</p>
```

Comment:
Hi, let us set defaultProps to make sure there is `0` displayed when there is no prop.
and also set propTypes to string